ui/ui-components/pages/destination/[id]/data.tsx

```diff
@@ -302,9 +302,8 @@ export default function Page(props) {
                       .map((group) => (
                         <option key={`grp-${group.id}`} value={group.id}>
                           {group.name}
-                          {process.env.GROUPAROO_UI_EDITION !== "config" && (
-                            <>&nbsp;{group.profilesCount} members</>
-                          )}
```

Comment:
cc @seancdavis

Reply:
Yikes! I guess I have to be better about testing this in Enterprise when I make a change like. I figured a simple conditional was simple enough.